src/python/WMCore/Database/DBCore.py

```diff
@@ -120,7 +127,6 @@ def connection(self):
         """
         return self.engine.connect()
 
-
```

Comment:
Please no extra whitespace movement (and squash the fixup of this).

Reply:
I will fix that.